### ISSUE TITLE
Make generated brew formulas `brew audit` compatible

### ISSUE THREAD

**Issue by lukasmalkmus** [goreleaser/goreleaser#1908]
**Is your feature request related to a problem? Please describe.**

I'm using `goreleaser` to release a Homebrew formula for my binary release. I'm using the `custom_block` config key to set a `license` field in the resulting formula as well. But when running `brew audit` on a formula generated by `goreleaser`, it yields some issues that could easily be omitted:

```
Trailing whitespace detected.
`license` (line 21) should be put before `bottle` (line 6)
```

**Describe the solution you'd like**

I propose two changes:

1. Just trim the trailing whitespaces. In theory, this should be a no brainer.
2. Instead of having the license set in the `custom_block`, make `license` a first class key in the `brews` config.

**Describe alternatives you've considered**

None

**Additional context**

None

---

I would love to contribute and make a PR if my issue sounds reasonable.

**Comment by caarlos0** [comment]
I remember trying fixing those issues, and then more issues would arise from the fact its not a valid brew formula (its a tap)... and the I just kind of gave up.

But I agree, license should be first class and removing the trailing whitespace should be easy enough to do too.

**Comment by lukasmalkmus** [comment]
Awesome, will figure out a PR on the weekend 👍

**Comment by lukasmalkmus** [comment]
Closed by #1911.

**Comment by caarlos0** [comment]
Many thanks @lukasmalkmus 🤘
